reference/configuration/framework.rst

```diff
@@ -347,7 +347,10 @@ need to escape the percent signs (``%``) by doubling them.
         // /path/to/guest/.../file will be opened
         // as /path/to/host/.../file on the host
         // and /foo/.../file as /bar/.../file also
-        'myide://%f:%l&/path/to/guest/>/path/to/host/&/foo/>/bar/&...'
+        'myide://%%f:%%l&/path/to/guest/>/path/to/host/&/foo/>/bar/&...'
+        
+        // example for PhpStorm
+        'phpstorm://open?file=%%f&line=%%l&/var/www/app/>/Users/greeflas/projects/my_project/'
```

Comment:
Let's use `/projects/my_project/` oid instead of prefixing it with your username? (to make the example more clear). We can then also update the foobar stuff at the lines above this, so the whole code block uses the same paths.

Reply:
@wouterj updated.